libraries/chain/include/steem/chain/smt_objects/smt_token_object.hpp

```diff
@@ -127,14 +126,11 @@ class smt_ico_object : public object< smt_ico_object_type, smt_ico_object >
    asset_symbol_type             symbol;
    steem::protocol::
    smt_capped_generation_policy  capped_generation_policy;
-   time_point_sec                generation_begin_time;
-   time_point_sec                generation_end_time;
-   time_point_sec                announced_launch_time;
-   time_point_sec                launch_expiration_time;
-
-   // smt_cap_reveal
-   share_type  steem_units_min_cap = -1;
-   share_type  steem_units_hard_cap = -1;
+   time_point_sec                contribution_begin_time;
+   time_point_sec                contribution_end_time;
+   time_point_sec                launch_time;
+   share_type                    steem_units_min_cap = -1;
```

Comment:
This name should have been rejected earlier. It should be `steem_units_soft_cap` to match soft/hard cap terminology in whitepaper.